articles/mfa/guides/mfa-api/phone.md

```diff
@@ -63,17 +69,17 @@ If successful, you'll receive a response like the one below:
 }
 ```
 
-If you get a `User is already enrolled error`, the user already has an MFA factor enrolled. Before associating another factor with the user, you need to challenge the user with the existing factor.
+If you get a `User is already enrolled error`, is because the user already has an MFA factor enrolled. Before associating it another factor, you need challenge the user with the existing one.
```

Comment:
I would change:
- `is because the user` to `it's because the user`
- `associating it another factor` to `associating it with another factor`